doc/sphinx/source/using.rst

```diff
@@ -234,6 +238,14 @@ pocl.
  contains all the intermediate compiler files are left as it is. This
  will be handy for debugging
 
+- **POCL_LEVEL0_JIT**
+
+ Sets up Just-In-Time compilation of Kernels (at clEnqueueNDRange time) in
```

Comment:
Please clarify what JIT means here (can mean a lot of things). The "eager" vs "lazy" compilation.